In [2]:
import numpy as np

x + y + z = 1328

x - y = - 120

x - z = 100

In [3]:
A = np.array([[1,1,1],[1,-1,0],[1,0,-1]])

In [4]:
A

array([[ 1,  1,  1],
       [ 1, -1,  0],
       [ 1,  0, -1]])

In [5]:
b = np.array([1328,-120,100])

In [6]:
b

array([1328, -120,  100])

In [8]:
np.linalg.solve(A,b)

array([436., 556., 336.])

Solution: 

436 items of iphone 6,

556 items of iphone 11,

336 items of iphone 12
